packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-node.vue

```diff
@@ -169,10 +170,11 @@ watch(
 			// Start 2nd simulation to get sample simulation from dill
 			const newInterventionResponse = await createInterventionPolicyFromOptimize(modelConfigId.value as string, optId);
 
-			const preForecastResponce = await startForecast(undefined);
-			const preForecastId = preForecastResponce.id;
-			const postForecastResponce = await startForecast(newInterventionResponse);
-			const postForecastId = postForecastResponce.id;
+			const preForecastResponce = startForecast(undefined);
+			const postForecastResponce = startForecast(newInterventionResponse);
+			const forescastResults = await Promise.all([preForecastResponce, postForecastResponce]);
```

Comment:
What is the purpose of wrapping these requests inside `Promise.all` ?
Also you have typos  `preForecastResponce` ==> preForecastPromise?